qa/wrapper.js

```diff
@@ -196,7 +242,47 @@ class OfficeDoc extends HTMLElement {
     doc.on('mouse_pointer', ({ payload }) => {
       embed.style.cursor = payload;
     });
-    // doc.on('comment', logit);
+
+    doc.on('clipboard_changed', async (response) => {
+      if (!response?.payload?.sw) {
+        return;
+      }
+      console.log('clipboard changed');
+
+      const clip = doc.getClipboard(['image/png', 'text/plain', 'text/html']);
+      // empty clipboard is likely an accidental clear from LOK
+      if (!clip.some((x) => x)) return;
+
+      const blobs = {};
+      for (const type of clip) {
+        if (!type) continue;
+
+        blobs[type.mimeType] = new Blob(
+          [type.mimeType === 'image/png' ? type.buffer : type.text],
+          { type: type.mimeType }
+        );
+      }
+      this.lastClipboardUpdate = Date.now().toString();
+      blobs['web text/x-macro-lastupdate'] = new Blob(
+        [this.lastClipboardUpdate],
+        {
+          type: 'web text/x-macro-lastupdate',
+        }
+      );
+      await navigator.clipboard.write([new ClipboardItem(blobs)]);
+    });
+
+    embed.addEventListener('keypress', async (e) => {
+      if ((e.code === 'KeyV' && e.ctrlKey) || e.metaKey) {
```

Comment:
Should this be 
```suggestion
      if (e.code === 'KeyV' && (e.ctrlKey || e.metaKey)) {
```

Reply:
It should, good catch. Prettier strikes again, but QA env mistakes shouldn't ever block things.